chenejac/TeslaRIS-backend
# Feature/basic additions

## Description

Added basic additionf for publisher, journal and conference entities. Added indexing for user and event entities. Fixed some bugs regarding registration logic as well as some previous persistence, indexing and reindexing errors.

## How should this be tested?
For every functionality as well as bugfixes there are (updated) unit and integration tests. You can verify each functionality manually by using client mocking tool e.g. Postman. For mail client, it is advised to use [mailhog](https://github.com/mailhog/MailHog).

## Checklist
